core/src/banking_stage/scheduler_messages.rs

```diff
@@ -0,0 +1,54 @@
+use {
+    crate::immutable_deserialized_packet::ImmutableDeserializedPacket,
+    solana_poh::poh_recorder::Slot, solana_sdk::transaction::SanitizedTransaction, std::sync::Arc,
+};
+
+/// A unique identifier for a transaction batch.
+#[derive(Clone, Copy, Debug, Hash, PartialEq, Eq)]
+pub struct TransactionBatchId(u64);
+
+impl TransactionBatchId {
+    pub fn new(index: u64) -> Self {
+        Self(index)
+    }
+}
+
+/// A unique identifier for a transaction.
+#[derive(Clone, Copy, Debug, Hash, PartialEq, Eq)]
+pub struct TransactionId(u64);
+
+impl TransactionId {
+    pub fn new(index: u64) -> Self {
+        Self(index)
+    }
+}
+
+/// Message: [Scheduler -> Worker]
+/// Transactions to be consumed (executed, recorded, committed)
```

Comment:
nit: `Transactions to be consumed (i.e. executed, recorded, and committed)`

Reply:
done: eb516c54f52d5eef56cf5e86f7886e89e6941380 + 61fb9aa7e85d409d9586fc310a2fbc903d2ed6ac